src/main/java/io/personium/core/model/jaxb/ObjectIo.java

```diff
@@ -63,7 +65,8 @@ private ObjectIo() {
             jsonContext = JAXBContextFactory.createContext(ObjectIo.class.getPackage().getName(),
                     ObjectIo.class.getClassLoader());
             nameSpaceToJsonMap.put("DAV:", "D");
-            nameSpaceToJsonMap.put("p:", "p");
+            //nameSpaceToJsonMap.put("p:", "p");
+            nameSpaceToJsonMap.put(CommonUtils.XmlConst.NS_PERSONIUM, CommonUtils.XmlConst.NS_PREFIX_PERSONIUM);
```

Comment:
This change causes the ACL data incompatibility. 
now the requireSchemaAuthz info is stored with key:
New Key; "@p.requireSchemaAuthz" 
instead of 
Ole Key: "@requireSchemaAuthz"